### ISSUE TITLE
Add OverWrite flag in reportOptions to archive reports

### ISSUE THREAD

**Issue by nischaytv** [LironEr/cypress-mochawesome-reporter#56]
**Is your feature request related to a problem? Please describe.**
I need to Archive my reports with timestamps, i tried "overwrite": false in reporterOptions but it didn't work. as it is removing output folder before running tests

![image](https://user-images.githubusercontent.com/20622111/152019467-05ab6e34-8bc8-49d6-9043-71c201dff954.png)

If you can stop removing output folder and control it with the help of overwrite flag, it will be great
![image](https://user-images.githubusercontent.com/20622111/152019708-fe2fd850-bd29-4a29-ae11-f5955e30d094.png)

**Comment by LironEr** [comment]
even with this option, every time the report will be generated it will have the same name so it will still override

I can suggest an alternative solution:
create `cypress.config.js` file

```js
module.exports = {
  reporter: 'cypress-mochawesome-reporter',
  video: false,
  reporterOptions: {
    reportDir: `reports/${new Date().toISOString()}`,
  },
};
```

run cypress:
```sh
cypress run --config-file cypress.config.js
```

This will create a new folder for each run

What do you think?

**Comment by LironEr** [comment]
oh got it, ignore my last comment.. as it can be a temporary solution.

I will fix overwrite flag behavior

**Comment by LironEr** [comment]
Fixed in [v2.4.0](https://github.com/LironEr/cypress-mochawesome-reporter/releases/tag/v2.4.0)
